src/storyMap/storyMapService.js

```diff
@@ -130,3 +145,123 @@ export const deleteStoryMap = ({ storyMap }) => {
     id: storyMap.id,
   });
 };
+
+export const addMemberships = ({ storyMap, emails, userRole }) => {
+  const query = graphql(`
+    mutation addMemberships(
+      $storyMapId: String!
+      $storyMapSlug: String!
+      $userEmails: [String!]!
+      $userRole: String!
+    ) {
+      saveStoryMapMembership(
+        input: {
+          storyMapId: $storyMapId
+          storyMapSlug: $storyMapSlug
+          userEmails: $userEmails
+          userRole: $userRole
+        }
+      ) {
+        memberships {
+          ...collaborationMembershipFields
+        }
+        errors
+      }
+    }
+  `);
+
+  return terrasoApi
+    .requestGraphQL(query, {
+      storyMapId: storyMap.storyMapId,
+      storyMapSlug: storyMap.slug,
+      userEmails: emails,
+      userRole,
+    })
```

Comment:
nit: in other mutations, we do this which has less code dup (applies to several mutations in this file):
```suggestion
  const query = graphql(`
    mutation addMemberships($input: StoryMapMembershipSaveMutationInput!) {
      saveStoryMapMembership(input: $input) {
        memberships {
          ...collaborationMembershipFields
        }
        errors
      }
    }
  `);

  return terrasoApi
    .requestGraphQL(query, {
      input: {
        storyMapId: storyMap.storyMapId,
        storyMapSlug: storyMap.slug,
        userEmails: emails,
        userRole,
      },
    })
```